bluetooth_mesh/mesh.py

```diff
@@ -105,25 +105,31 @@ class AccessMessage:
     MAX_TRANSPORT_PDU = 15
     SEGMENT_SIZE = 12
 
-    def __init__(self, src, dst, ttl, payload):
+    def __init__(self, src, dst, ttl, payload, szmic=False):
```

Comment:
Please move the `szmic` argument into `transport_pdu` method, and remove `self.szmic`